docs/user/devops-pipelines.md

```diff
@@ -52,3 +51,78 @@ will be copied over**_. Values must be set on the next Stage's Instance explicit
 **Instance Settings** - None of your Instance Settings will be copied over (e.g. Editor, Palette or Security Settings).
 
 This ensures a split between your staging environments.
+
+## Pipeline Stage details
+
+* Stages of a pipeline are executed from the selected stage to the next stage immediately to the right.
```

Comment:
I originally had that @joepavitt. I feel it infers that _any_ stage on the left can affect any upstream stage on the right.  My re-write is technically more accurate (if somewhat less succinct).  

Skipping this suggestion unless otherwise contested

Reply:
So I can move forward, I will apply suggestion.